openmetadata-ui/src/main/resources/ui/src/pages/Glossary/GlossaryPage/GlossaryPage.component.tsx

```diff
@@ -242,23 +249,23 @@ const GlossaryPage = () => {
     );
   }
 
-  if (isLoading || isUndefined(selectedData)) {
-    return <Loader />;
-  }
-
   return (
     <PageContainerV1>
       <PageLayoutV1 leftPanel={<GlossaryLeftPanel glossaries={glossaries} />}>
-        <GlossaryV1
-          deleteStatus={deleteStatus}
-          handleGlossaryTermUpdate={handleGlossaryTermUpdate}
-          isChildLoading={isRightPanelLoading}
-          isGlossaryActive={isGlossaryActive}
-          selectedData={selectedData}
-          updateGlossary={updateGlossary}
-          onGlossaryDelete={handleGlossaryDelete}
-          onGlossaryTermDelete={handleGlossaryTermDelete}
-        />
+        {isUndefined(selectedData) || isRightPanelLoading ? (
```

Comment:
Loader should only be depend on `isRightPanelLoading`

Reply:
updated